js/main.js

```diff
@@ -0,0 +1,23 @@
+const getRandomNumber = (a, b) => {
+  if (a<0 || b<0) {
```

Comment:
Не забывай ставить между операндами пробелы
```suggestion
  if (a < 0 || b < 0) {
```